### ISSUE TITLE
Using 'ref:' to declare variables starting with 'in' will cause the extension to crash

### ISSUE THREAD

**Issue by zerowong** [johnsoncodehk/volar#85]
![image](https://user-images.githubusercontent.com/31186725/110031392-d0a23580-7d71-11eb-86d4-835ca98940e4.png)

The extension started reporting errors and crashing the moment I pressed the 'n' key. I could only stop it by restarting Vscode.

Console ouput:
Volar - API:
```
[Error - 上午5:23:17] Request completionItem/resolve failed.
  Message: Request completionItem/resolve failed with message: Invalid count value
  Code: -32603 
```
Volar - Document:
```
(node:14904) UnhandledPromiseRejectionWarning: RangeError: Invalid count value
    at String.repeat (<anonymous>)
    at writeScriptSetup (c:\Users\NEO\.vscode\extensions\johnsoncodehk.volar-0.21.20\node_modules\@volar\vscode-vue-languageservice\out\generators\script_suggestion.js:42:88)
    at Object.generate (c:\Users\NEO\.vscode\extensions\johnsoncodehk.volar-0.21.20\node_modules\@volar\vscode-vue-languageservice\out\generators\script_suggestion.js:12:5)
    at c:\Users\NEO\.vscode\extensions\johnsoncodehk.volar-0.21.20\node_modules\@volar\vscode-vue-languageservice\out\virtuals\script.js:22:83
    at ComputedRefImpl.reactiveEffect [as effect] (c:\Users\NEO\.vscode\extensions\johnsoncodehk.volar-0.21.20\node_modules\@vue\reactivity\dist\reactivity.cjs.js:46:24)
    at ComputedRefImpl.get value [as value] (c:\Users\NEO\.vscode\extensions\johnsoncodehk.volar-0.21.20\node_modules\@vue\reactivity\dist\reactivity.cjs.js:830:32)
    at c:\Users\NEO\.vscode\extensions\johnsoncodehk.volar-0.21.20\node_modules\@volar\vscode-vue-languageservice\out\virtuals\script.js:32:36
    at ComputedRefImpl.reactiveEffect [as effect] (c:\Users\NEO\.vscode\extensions\johnsoncodehk.volar-0.21.20\node_modules\@vue\reactivity\dist\reactivity.cjs.js:46:24)
    at ComputedRefImpl.get value [as value] (c:\Users\NEO\.vscode\extensions\johnsoncodehk.volar-0.21.20\node_modules\@vue\reactivity\dist\reactivity.cjs.js:830:32)
    at c:\Users\NEO\.vscode\extensions\johnsoncodehk.volar-0.21.20\node_modules\@volar\vscode-vue-languageservice\out\sourceFile.js:110:56
```
Volar - HTML:
```
[Error - 上午5:23:17] Request textDocument/foldingRange failed.
  Message: Request textDocument/foldingRange failed with message: Invalid count value
  Code: -32603 
```
Vscode version: 1.53.2
Volar version: 0.21.20

**Comment by johnsoncodehk** [comment]
Confirmed. Thanks to report!

**Comment by johnsoncodehk** [comment]
Fixed in 0.22.0.
